packages/client/src/pages/Game/game.css

```diff
@@ -7,3 +7,27 @@
   image-rendering: -moz-crisp-edges;
   image-rendering: -webkit-crisp-edges;
 }
+
+.game__label-game-over {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  margin: auto;
+  margin-top: 300px;
+  color: white;
+  font-size: 30px;
+}
+
+.game__button-game-over {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  margin: auto;
+  margin-top: 500px;
+  width: 200px;
```

Comment:
```suggestion
  width: 250px;
```

Мне кажется, можно чуть изменить тут
![image](https://user-images.githubusercontent.com/106176669/204808395-e42fbd5e-86e3-4580-a4c8-9c62550d957e.png)